deploy.sh

```diff
@@ -60,6 +60,13 @@ else
     fi
 fi
 
+aws sts get-caller-identity
+result=$?
+if [ $result != 0 ]; then
+    echo "AWS ACCESS KEY or SECRET isn't set. Check out our doc to understand what you're missing https://bitops.sh/examples/#docker-run-examples"
```

Comment:
A more detailed error message and mentioning the ENV variables explicitly would be more helpful for someone hitting this error: 
```suggestion
    echo "AWS_ACCESS_KEY_ID or AWS_SECRET_ACCESS_KEY required for AWS authentication are not set or invalid."
    echo "Check out BitOps documentation to understand what you're missing https://bitops.sh/examples/#docker-run-examples"
```